File: Java/lab10/src/main/java/org/example/ServiceEmployee.java
package org.example;

public class ServiceEmployee extends WorkerDecorator {
    public ServiceEmployee(Worker worker) {
        super(worker);
    }
    public String makeJob(){
        return " Repair devices.";
    }
    @Override
    public String doWork() {
        return super.doWork()+makeJob();
    }
}

File: Python/lab10/ServiceEmployee.py
from Worker import Worker
from WorkerDecorator import WorkerDecorator

class ServiceEmployee(WorkerDecorator):
    def __init__(self, worker: Worker):
        super().__init__(worker)

    def makeJob(self) -> str:
        return " Repair devices."

    def doWork(self) -> str:
        return super().doWork() + self.makeJob()
